File: src/main/java/com/baulsupp/oksocial/services/surveymonkey/SurveyMonkeyUtil.java
package com.baulsupp.oksocial.services.surveymonkey;

import com.google.common.collect.Sets;
import java.util.Collections;
import java.util.Set;
import okhttp3.Request;

public class SurveyMonkeyUtil {
  private SurveyMonkeyUtil() {
  }

  public static final Set<String> API_HOSTS =
      Collections.unmodifiableSet(Sets.newHashSet(
          "api.surveymonkey.net")
      );

  public static Request apiRequest(String s, Request.Builder requestBuilder) {
    return requestBuilder.url("https://api.surveymonkey.net" + s).build();
  }
}

File: src/main/kotlin/com/baulsupp/oksocial/services/surveymonkey/SurveyMonkeyUtil.kt
package com.baulsupp.oksocial.services.surveymonkey

import okhttp3.Request

object SurveyMonkeyUtil {

    val API_HOSTS = setOf((
            "api.surveymonkey.net")
    )

    fun apiRequest(s: String, requestBuilder: Request.Builder): Request {
        return requestBuilder.url("https://api.surveymonkey.net" + s).build()
    }
}
